lib/services/extensions-service-base.ts

```diff
@@ -52,7 +52,7 @@ export class ExtensionsServiceBase {
 
 			if( this.shouldUpdatePackage(cachedVersion, extensionVersion) ) {
 				this.$logger.printInfoMessageOnSameLine(`Updating ${packageName} package...`);
-				let zipFileName = temp.path({ path:  path.join(this.cacheDir, packageName + ".zip") });
+				let zipFileName = path.join(this.cacheDir, packageName + ".zip");
```

Comment:
Due to `temp` module not being used anymore, there is a chance that this file is not deleted if an exception is raised during unzip for example